.github/workflows/ci.yaml

```diff
@@ -57,6 +57,7 @@ jobs:
         run: |
           cd build/clang-cxx17/
           cp -Rp ./installed python-${{ env.version_in_development }}-macos-x86_64
+          cp install_certificates.py python-${{ env.version_in_development }}-macos-x86_64/
```

Comment:
This file is in fact unneeded when the openssl library linked to point to /etc/ssl as default dir, macOS get that directory provisioned and configured on first boot.

I keep it in the package but tipi doesn't use it anymore since we have : 
[567dd79](https://github.com/tipi-deps/adapter-cpython/pull/1/commits/567dd795835a5393d2e09548f23c468cb421c938)